.github/paths-labeller.yml

```diff
@@ -0,0 +1,53 @@
+---
+- "beats":
+  - "detection_rules/beats.py"
+  - "etc/beats_schemas/**/*"
+- "documentation":
+  - "./**/*.md"
+- "Domain: Cloud":
+  - "rules/integrations/aws/**/*.toml"
+  - "rules/integrations/azure/**/*.toml"
+  - "rules/integrations/gcp/**/*.toml"
+  - "rules/integrations/google-workspace/**/*.toml"
+  - "rules/integrations/microsoft-365/**/*.toml"
+  - "rules/integrations/okta/**/*.toml"
+- "Domain: Endpoint":
+  - "rules/windows/**/*.toml"
+  - "rules/linux/**/*.toml"
+  - "rules/macos/**/*.toml"
+- "ecs":
+  - "detection_rules/ecs.py"
+  - "etc/ecs_schemas/**/*"
+- "ML":
+  - "rules/ml/**/*"
+- "OS: Linux":
+  - "rules/linux/**/*.toml"
+- "OS: macOS":
+  - "rules/macos/**/*.toml"
+- "OS: Windows":
+  - "rules/windows/**/*.toml"
+- "Platform: AWS":
+  - "rules/integrations/aws/**/*.toml"
+- "Platform: Azure":
+  - "rules/integrations/azure/**/*.toml"
+- "Platform: Crowdstrike":
+  - "rules/integrations/crowdstrike/**/*.toml"
+- "Platform: CyberArkPas":
+  - "rules/integrations/cyberarkpas/**/*.toml"
+- "Platform: GCP":
+  - "rules/integrations/gcp/**/*.toml"
+- "Platform: Google Workspace":
+  - "rules/integrations/google-workspace/**/*.toml"
+- "Platform: Microsoft 365":
+  - "rules/integrations/microsoft-365/**/*.toml"
+- "Platform: Okta":
+  - "rules/integrations/okta/**/*.toml"
+- "python":
+  - "detection_rules/**/*.py"
```

Comment:
what about KQL and other modules?

Reply:
created `kibana-module` label and created entries:

```
- "kibana-module":
  - "kibana/**/*"
- "kql":
  - "kql/**/*"
```

Reply:
oh i meant... i would rather not have module specific labels and would just call them all "python", regardless of where they are

Reply:
oh... so like it was? :nick-young-v2:

Reply:
detection_rules/**
kibana/**
kql/**

etc